Directory.Build.props

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
NIT: looks like this is changing the encoding of this file, which is just going to cause a subsequent PR to change this again. Can we try to undo the encoding change?

Reply:
Sure, and yes I am still working on it.

Reply:
Often this is caused by Notepad adding a byte order mark.

Reply:
As well as improperly rendering files with mac or linux style endlines which then forces a guess where to break the lines. If only it was made to not care like what notepad++ does.

Reply:
They are supposed to have fixed the line endings recently. Anyway I don't recommend it as a code editor... Maybe VS Code!

Reply:
@joperezr I invited you to the repository so you can make changes to it.

Reply:
@danmosemsft is it alright if I open a pull request that is a prerequisite to this one that fixes the xml doc issues and then rerun the build based on that change to see if build fixes itself then? Or should I file a separate issue to fix them that is then a prerequisite of the issue I am fixing with this pull and to this pull as well?

Reply:
It is OK to suppress warnings per project or if very common directly in Directory.Build.props where you're already suppressing some common ones. Just make sure to add a comment wherever you suppress them to the issue you just opened.  That way when you work on the issue (or someone else) we can just grep the issue number and see we don't miss any project that needs to be fixed.

Reply:
@AraHaan if you are asking what order to work in, either is fine.

Reply:
Alright.